sympy/printing/tests/test_latex.py

```diff
@@ -213,6 +213,8 @@ def test_latex_SingularityFunction():
         r"{\left\langle x - 4 \right\rangle}^{-2}"
     assert latex(SingularityFunction(x, 4, -1)) == \
         r"{\left\langle x - 4 \right\rangle}^{-1}"
+    assert latex(SingularityFunction(x, 0, 0)**2) == \
+        r"{\\left\\langle x \\right\\rangle}^{0}"
```

Comment:
The r in front of the string means that you do not have to escape `\`, so either remove the r or replace `\\` with `\`.